config/routes.rb

```diff
@@ -30,6 +30,9 @@
     put '/gestionnaires' => 'gestionnaires/registrations#update', :as => 'gestionnaires_registration'
   end
 
+  get 'avis/:id/sign_up/email/:email' => 'backoffice/avis#sign_up', constraints: { email: /.*/ }, as: 'avis_sign_up'
```

Comment:
Ça permet de ne pas exposer les emails des invités si quelqu'un itère sur l'`avis_id`.